<launch>
  <!-- Arm Motion -->
  <arg name="arm_movement_service" default="move_arm" />

  <arg name="arm_movement_control_service" default="control_arm" />

  <node pkg="tangible_programming" type="arm_motion_node" name="arm_motion_node"> 
    <param name="arm_movement_service" type="string" value="$(arg arm_movement_service)" />
    <param name="arm_movement_control_service" type="string" value="$(arg arm_movement_control_service)" />
  </node>

</launch>